integrations/linear/schema.zod.ts

```
@@ -1,5 +1,6 @@
 // Generated by ts-to-zod
 import { z } from 'zod';
+import { type Field } from './../.nango/schema';
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)